scripts/update_secrets.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"regexp"
+)
+
+// Check if all the rules that exist in "gitleaks" are included in our list of rules (in secret.go file)
+func main() {
+
+	//Find the latest release of "gitleaks"
+	release, err := fetchLatestRelease()
+	if err != nil {
+		fmt.Printf("Error fetching latest release: %s\n", err)
+		return
+	}
```

Comment:
Do it for all the errors
```suggestion
	if err != nil {
		fmt.Printf("Error fetching latest release: %s\n", err)
		os.Exit(1)
	}
```